### PR TITLE
switch to PS handling target size

### PR DESCRIPTION
**Pull request by pineapplespatula**
code to pass target Width/Height to PS for handling .

### CODE REVIEW

**Comment by pineapplespatula** [comment]
requires https://github.com/adobe-photoshop/generator-assets/pull/412

File: lib/generator.js [hunk 1]
```diff
@@ -1287,9 +1289,11 @@
      * @param {!Object<String,float>} staticInputBounds  Bounds for the user-provided content (pixels, shapes)
      * @param {!Object<String,float>} visibleInputBounds Bounds for the visible content (user-provided + effects)
      * @param {!Object<String,float>} paddedInputBounds  Bounds for the whole image (visible + padding)
+     * @param {boolean=} clipToBounds if we should clip this object to the document or artboard bounds
+     * @param {Object=} layerComp getPixmap settings
```
Comment (mcilroyc) [inline lib/generator.js]: Seems like `layerComp` is just used as a boolean flag, right?

File: lib/generator.js [hunk 2]
```diff
@@ -990,6 +990,8 @@
                 outputRect: settings.outputRect,
                 scaleX:     settings.scaleX || 1,
                 scaleY:     settings.scaleY || 1,
+                targetWidth: settings.targetWidth,
+                targetHeight: settings.targetHeight,
```
Comment (mcilroyc) [inline lib/generator.js]: If you supply only one of these target values, does PS do a reasonable thing?  See also: these setting need to be in JSDoc somewhere

**Comment by mcilroyc** [comment]
Also what's the minimum jenkins build for this?

**Comment by pineapplespatula** [comment]
Jenkins build 323 is required

**Comment by mcilroyc** [comment]
back to @pineapplespatula: JSDoc nit, layerComp-as-boolean nit, and a question.

File: lib/generator.js (hunk 2 above)
Comment (pineapplespatula) [inline lib/generator.js]: yes PS does the right thing with only one value

**Comment by mcilroyc** [comment]
@pineapplespatula  please prepare a merge to main that includes whatever changes currently in the generator branch necessary to support this PR

**Pull request merged**